airflow/models/dagpickle.py

```diff
@@ -39,7 +39,7 @@ class DagPickle(Base):
     id = Column(Integer, primary_key=True)
     pickle = Column(PickleType(pickler=dill))
     created_dttm = Column(UtcDateTime, default=timezone.utcnow)
-    pickle_hash = Column(Text)
+    pickle_hash = Column(BIGINT)
```

Comment:
Have you checked what value we right on this column?

Reply:
It's `BigInteger`: https://github.com/apache/airflow/blob/d19cb86660d40e665d8c4fe2b07d76b88532bd8b/airflow/migrations/versions/0001_1_5_0_current_schema.py#L83
There's no other place that it was changed.

I will update it to use `BigInteger` instead of `BIGINT`